deployments/cognito-rds-s3/terraform/main.tf

```diff
@@ -182,10 +204,19 @@ module "eks_blueprints_kubernetes_addons" {
       }
     ]
   }
+  
   enable_secrets_store_csi_driver_provider_aws = true
 
   tags = local.tags
+}
 
+module "eks_data_addons" {
+  source = "aws-ia/eks-data-addons/aws"
+  version = "~> 1.0" # ensure to update this to the latest/desired version
+
+  oidc_provider_arn = module.eks_blueprints.eks_oidc_provider_arn
```

Comment:
dont need to pass cluster endpoint etc? like for aws-ia/eks-blueprints-addons/aws

Reply:
Seems to work without it, but ill include it in case.

Reply:
Looks like there is no option to pass it in https://github.com/aws-ia/terraform-aws-eks-data-addons and https://github.com/aws-ia/terraform-aws-eks-data-addons/blob/main/variables.tf